### PR TITLE
AP_Scripting: Change to ArduPilot coding style

### PR DESCRIPTION
**Pull request by muramura**
I think it is better to combine the minimum coding style.

http://ardupilot.org/dev/docs/style-guide.html

### CODE REVIEW

File: libraries/AP_Scripting/lua_generated_bindings.cpp [hunk 1]
```diff
@@ -13,7 +13,8 @@
 #include <AP_Common/Location.h>
 
 
-static int binding_argcheck(lua_State *L, int expected_arg_count) {
+static int binding_argcheck(lua_State *L, int expected_arg_count)
```
Comment (WickedShell) [inline libraries/AP_Scripting/lua_generated_bindings.cpp]: You can't edit this file, you must edit the generator if you want to make a change.

Comment (muramura) [reply libraries/AP_Scripting/lua_generated_bindings.cpp]: @WickedShell san.   thank you for your comment.
I adapted the generator to the ArduPilot coding style.

File: libraries/AP_Scripting/generator/Makefile [hunk 2]
```diff
@@ -13,7 +13,7 @@ OBJS = $(patsubst %,$(OBJ_DIR)/%,$(_OBJS))
 
 .PHONY: all
 all: directories $(OBJS)
-	$(CC) $(LDFLAGS) $(OBJS) -o $(OUT)
+	$(CC) $(OBJS) -o $(OUT) $(LDFLAGS)
```
Comment (magicrub) [inline libraries/AP_Scripting/generator/Makefile]: Does this change any behavior?

Comment (muramura) [reply libraries/AP_Scripting/generator/Makefile]: @magicrub san.   I got an error in the build. It was able to build normally if it changed.
====
This change:
muramura@Aspire:~/work/ardupilot/libraries/AP_Scripting/generator$ make
mkdir -p build
mkdir -p build/obj
gcc -Wall -Werror -Wextra -g -c src/main.c -o build/obj/main.c.o
gcc build/obj/main.c.o -o build/gen-bindings -lreadline
cp build/gen-bindings .

====
before:
muramura@Aspire:~/work/ardupilot/libraries/AP_Scripting/generator$ make
mkdir -p build
mkdir -p build/obj
gcc -Wall -Werror -Wextra -g -c src/main.c -o build/obj/main.c.o
gcc -lreadline build/obj/main.c.o -o build/gen-bindings 
build/obj/main.c.o: In function `start_line':
/home/muramura/work/ardupilot/libraries/AP_Scripting/generator/src/main.c:198: undefined reference to `readline'
collect2: error: ld returned 1 exit status
Makefile:16: recipe for target 'all' failed
make: *** [all] Error 1

**Comment by WickedShell** [comment]
Closing as it's a mostly a noise PR that serves to generate conflicts with in progress work, that doesn't add much value. (Particularly since the whole files are in that style it is at least consistent)

**Pull request closed without merging**